### PR TITLE
feat(admin-ui): impl short codes in webhooks to pass dynamic data #1542

### LINKED ISSUE #1542: feat(admin-ui): implement short codes in webhooks to pass dynamic info on webhook execution
- Add `short codes` in webhook url & request body when creating/updating a webhook.
- Trigger webhook & replace the short codes from url/request body using the success response of an event.

Parent ticket# https://github.com/GluuFederation/flex/issues/1503

Screenshot 2024-01-17 at 11 46 33 AM

### PR DESCRIPTION
**Pull request by jv18creator**
closes #1542

### CODE REVIEW

File: admin-ui/plugins/admin/components/Webhook/ShortcodePopover.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import { List, ListItemButton, ListItemText } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import { HelpOutline } from '@mui/icons-material'
+import { Tooltip as ReactTooltip } from 'react-tooltip'
+import PropTypes from 'prop-types'
+
+const Icon = () => (
+  <svg
```
Comment (duttarnab) [inline admin-ui/plugins/admin/components/Webhook/ShortcodePopover.js:14]: Can we keep images and CSS separately

File: admin-ui/plugins/admin/components/Webhook/ShortcodePopover.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  }
+
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  const open = Boolean(anchorEl)
+  const id = open ? 'simple-popover' : undefined
+
+  const wrapperStyles = {
```
Comment (duttarnab) [inline admin-ui/plugins/admin/components/Webhook/ShortcodePopover.js:47]: Can we keep images and CSS separately

File: admin-ui/plugins/admin/helper/ShortCodes.json [hunk 3]
```diff
@@ -0,0 +1,1123 @@
+{
```
Comment (duttarnab) [inline admin-ui/plugins/admin/helper/ShortCodes.json:1]: Please rename the file to shortCodes.json

File: admin-ui/plugins/admin/components/Webhook/ShortcodePopover.js (hunk 2 above)
Comment (jv18creator) [reply admin-ui/plugins/admin/components/Webhook/ShortcodePopover.js]: css moved to `app/routes/Apps/Gluu/styles/applicationstyle.js`

**Review by duttarnab — APPROVED** [review]

**Pull request merged**